Markdown changes (#377)
sirgwain/craig-stars: README.md

![screenshot](docs/screenshots/screenshot1.png)

An open-source, web based Stars! clone/mobile port.

[![Build Status](https://github.com/sirgwain/craig-stars/actions/workflows/build.yml/badge.svg)](https://github.com/sirgwain/craig-stars/actions/workflows/build.yml)

## About

> [Stars!](https://wikipedia.org/en/Stars!) is an interstellar, turn based strategy game and a member of the 4X genre (eXplore, eXpand, eXploit, eXterminate). The original Stars! game was released as shareware by Jeff Johnson and Jeff McBride with help from Jeffrey Krauss (collectively known as the Jeffs), with a retail version published in 1997. `craig-stars` attempts to remain as close to the original as possible in both UI and gameplay, while also adding functionality and allowing it to function on mobile devices.

Reviewer: Not sure I like this statement in the README professing mobile support. Now people will _expect_ it to work! 😀

Author: I mean it _does_ work quite well tbf...

> The [Stars! wiki](https://wiki.starsautohost.org/) and [Home World Forum](https://starsautohost.org/sahforum2/) are both great places for learning about Stars! gameplay. The wiki's [strategy guide](https://wiki.starsautohost.org/wikinew/ssg/ssg.htm) is a good starting point for new players to learn the basics, and the active [Stars! discord server](https://discord.gg/t8JpDS3d) is a great place to ask questions and find new games every once in a while. (For developers & contributors, it also has a channel for stars clone discussion.)

Reviewer: ```suggestion
The [Stars! wiki](https://wiki.starsautohost.org/) and [Home World Forum](https://starsautohost.org/sahforum2/) are both great places for learning about Stars! gameplay. The wiki's [strategy guide](https://wiki.starsautohost.org/wikinew/ssg/ssg.htm) is a good starting point for new players to learn the basics, and the active [Stars! discord server](https://discord.gg/t8JpDS3d) is a great place to ask questions and start new multiplayer games. (For developers & contributors, there is also has a channel for stars clone discussion.)
```

## Development/Contributing

> Contributions of any form are most certainly welcome, even non-code ones such as bug reports & feature requests. If you have any issues to report or cool things to implement, open an Issue report on the GitHub repository and someone will ~~eventually~~ get back to you. 

Reviewer: ```suggestion
Contributions of any form are welcome, even non-code ones such as bug reports & feature requests. If you have any issues to report or cool ideas to suggest, open an Issue report on the GitHub repository. 
```

> For code contributions, all you need to get started is a good code editor (like Visual Studio Code - avaliable at [https://code.visualstudio.com/Download](https://code.visualstudio.com/Download) or online at [https://vscode.dev](https://vscode.dev)), a GitHub account and a computer wih internet access. (Having a brain or prior programming knowledge certainly doesn't hurt, but it's nothing a good web tutorial or two won't fix.) 

Reviewer: ```suggestion
For code contributions, all you need to get started is a good code editor (like Visual Studio Code - avaliable at [https://code.visualstudio.com/Download](https://code.visualstudio.com/Download) or online at [https://vscode.dev](https://vscode.dev)), and a GitHub account.
```
